chart/values.yaml

```diff
@@ -1,8 +1,12 @@
-mayastorImagesTag: develop
+mayastorImagesTag: latest
 mayastorImagePullPolicy: Always
-mayastorCpuCount: "2"
+mayastorCpuCount: "1"
```

Comment:
I don't know exactly how this is supposed to work but.. how does this get filled in the daemon set values?

Reply:
these values replace placeholders in chart/templates and the result is saved to the deploy directory. In a nutshell -  just a simplified explanation.